client/src/containers/analysis-sidebar/component.tsx

```diff
@@ -14,10 +14,12 @@ import {
 } from 'store/features/analysis/scenarios';
 import { useInfiniteScenarios } from 'hooks/scenarios';
 import useBottomScrollListener from 'hooks/scroll';
+import { usePermissions } from 'hooks/permissions';
 import ScenariosFilters from 'containers/scenarios/filters';
 import { Anchor } from 'components/button';
 import Loading from 'components/loading';
 import ScenarioItem from 'containers/scenarios/item';
+import { Permission } from 'hooks/permissions/enums';
```

Comment:
```suggestion
import type { Permission } from 'hooks/permissions/enums';
```

Reply:
@davidsingal we did not put a type in here because it is an `enum`, not a type. And the srict mode was telling us that setting a type there is an error.